service/src/main/java/uk/gov/hmcts/reform/fpl/service/DraftCMOService.java

```diff
@@ -87,6 +88,7 @@ public void removeTransientObjectsFromCaseData(Map<String, Object> caseData) {
     public DynamicList buildDynamicListFromHearingDetails(List<Element<HearingBooking>> hearingDetails) {
         List<HearingDateDynamicElement> hearingDates = hearingDetails
             .stream()
+            .filter(hearingBooking -> hearingBooking.getValue().getStartDate().isAfter(LocalDateTime.now()))
```

Comment:
I am nervous we operate on LocalDateTimes, but I think it will work in this instance. At least I can't figure out scenario where it's broken. But remember - the LocalDateTime on machine is UTC which is diifferent than during daylightsaving time.

Reply:
Would the time configuration that was created a while ago help solve this potential problem?

Reply:
Hmm, no this may be a problem. If a hearing is booking during daylight saving time, then the check would be an hour out? so it would remove hearing dates prematurely?

Reply:
I don't think it would, the backend would always store this as UTC and treat LocalDateTime as UTC, so date specified in UI would be converted to UTC by CCD frontend and we would treat is then as LocalDateTime, because servers are always UTC I think it would work. As long as we don't display the date to the user it's fine. This is why we need to do date transformation to UK when we display it, but yeah we probably should test it a bit.